File: Back End/file-upload-service/src/main/java/com/chamoddulanjana/fileuploadservice/dto/VehicleDto.java
package com.chamoddulanjana.fileuploadservice.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;


@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
public class VehicleDto {
    private String id;
    private String firstName;
    private String lastName;
    private String carMake;
    private String carModel;
    private String vin;
    private Date manufacturedDate;
}

File: NestJS-BackEnd/file-uploads/src/vehicle/dto/model/vehicleDto.ts
import {Field, Int, ObjectType} from "@nestjs/graphql";

@ObjectType()
export class VehicleDto {

    @Field(() => Int, { description: 'Example field (placeholder)' })
    id: number;

    @Field(() => String)
    first_name: string;

    @Field(() => String)
    last_name: string;

    @Field(() => String)
    email: string;

    @Field(() => String)
    car_make: string;

    @Field(() => String)
    car_model: string;

    @Field(() => String)
    vin: string;

    @Field(() => Date)
    manufactured_date: Date;

    @Field(() => Int, { nullable:true})
    age_of_vehicle: number;
}